2.extract-features/2.cp-feature-extraction-env.yml

```diff
@@ -0,0 +1,10 @@
+name: 2.cp-feature-extraction-cell-health
```

Comment:
Consider using the project name as a prefix for the environment name which might assist with the ordering when viewing many projects and environments within the same system. For example: `cell-health-data_2.cp-feature-extraction` (or similar).

Reply:
I think that change would be best left for another PR as I am following the environment name precedent set in the previous modules of this repo.